src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,419 @@
+.pf-c-wizard {
+  --pf-c-wizard--GridTemplateRows: auto auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateRows: auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateColumns: 300px 1fr;
+  --pf-c-wizard--Width: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--Height: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-wizard--children--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-wizard--GridTemplateRows: var(--pf-c-wizard--md--GridTemplateRows);
+  }
+
+  // Header
+  --pf-c-wizard__header--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__header--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Close
+  --pf-c-wizard__close--Top: calc(var(--pf-global--spacer--lg) - var(--pf-global--spacer--form-element));
+  --pf-c-wizard__close--Right: var(--pf-global--spacer--md);
+  --pf-c-wizard__close--FontSize: var(--pf-global--FontSize--xl);
+
+  // Title
+  --pf-c-wizard__title--PaddingRight: var(--pf-global--spacer--xl);
+
+  // Description
+  --pf-c-wizard__description--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-wizard__description--Color: var(--pf-global--Color--400);
+
+  // Steps
+  --pf-c-wizard__steps--md--BoxShadow: var(--pf-global--BoxShadow--md-right);
+
+  // Toggle
+  --pf-c-wizard__toggle--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__toggle--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingLeft: calc(var(--pf-global--spacer--xl) + var(--pf-c-wizard__nav-link--before--Width) + var(--pf-global--spacer--sm));
+  --pf-c-wizard__toggle--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+  --pf-c-wizard__toggle--ZIndex: var(--pf-global--ZIndex--sm);
+
+  // Toggle List
+  --pf-c-wizard__toggle-list--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-wizard__toggle-list--MarginBottom: calc(var(--pf-c-wizard__toggle-list-item--MarginBottom) * -1);
+
+  // Toggle List Item
+  --pf-c-wizard__toggle-list-item--not-first-child--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-wizard__toggle-list-item--not-last-child--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-wizard__toggle-list-item--MarginBottom: var(--pf-global--spacer--xs);
+
+  // Toggle Number
+  --pf-c-wizard__toggle-num--MarginRight: var(--pf-global--spacer--sm);
+
+  // Toggle Divider
+  --pf-c-wizard__toggle-divider--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Toggle Icon
+  --pf-c-wizard__toggle-icon--MarginTop: #{pf-size-prem(3px)};
+
+  // Nav
+  --pf-c-wizard__nav--Top: 100%;
+  --pf-c-wizard__nav--MaxHeight: #{pf-size-prem(200px)};
```

Comment:
Looks like the max height is larger in the invision file and has more padding

<img width="1010" alt="screen shot 2019-02-18 at 11 02 28 am" src="https://user-images.githubusercontent.com/20118816/52963039-c4a2d100-336c-11e9-9892-6ac83a30eeff.png">

Reply:
Talked to @kybaker and just allowing this to consume the available area when the nav is open.